libs/jsonforms-components/project.json

```diff
@@ -12,26 +12,6 @@
         "lintFilePatterns": ["libs/jsonforms-components/**/*.{ts,tsx,js,jsx}"]
       }
     },
-    "build": {
-      "executor": "@nrwl/rollup:rollup",
-      "outputs": ["{options.outputPath}"],
-      "options": {
-        "outputPath": "dist/libs/jsonforms-components",
-        "tsConfig": "libs/jsonforms-components/tsconfig.lib.json",
-        "project": "libs/jsonforms-components/package.json",
-        "entryFile": "libs/jsonforms-components/src/index.ts",
-        "external": ["react/jsx-runtime"],
-        "rollupConfig": "@nrwl/react/plugins/bundle-rollup",
-        "compiler": "babel",
```

Comment:
Hi @jonathanweyermann, do you have any specific reason to remove the build here.

Reply:
yeah, it caused a linting error - https://stackoverflow.com/questions/74887933/buildable-libraries-cannot-import-or-export-from-non-buildable-libraries-eslint